common/app/common/LinkTo.scala

```diff
@@ -20,19 +20,17 @@ trait LinkTo extends Logging {
 
   lazy val host = Configuration.site.host
 
-  private val ProdURL = "^http://www.theguardian.com/.*$".r
-  private val GuardianUrl = "^(http://www.theguardian.com)?(/.*)?$".r
+  private val ProdURL = "^http[s]?://www.theguardian.com/.*$".r
+  private val GuardianUrl = "^(http[s]?://www.theguardian.com)?(/.*)?$".r
```

Comment:
actually (s)? should be enough, square is for a set of characters wheras () is for grouping